samples/boards/nrf/nrf53_sync_rtc/README.rst

```diff
@@ -0,0 +1,76 @@
+.. _nrf53_sync_rtc_sample:
+
+Synchronized RTC
+################
+
+Overview
+********
+
+Sample is showing how RTC clocks used for system clock on application and network
+core are synchronized. The result of synchronization is an offset value on network
+core which can be applied to the system tick for logging timestamping.
+
+Sample is using IPM driver and IPC HAL to produce events which occur at same time on
+both cores. Application core periodically reads current system tick and stores it in
+the shared memory and triggers IPC task which results in the interrupt on the network
+core. In the context of the IPC interrupt handler, network core is logging timestamp
+from shared memory and local system tick updated by the offset. User can observe
+that before synchronization procedure is completed, timestamps differ significantly
+and once procedure is completed timestamps are synchronized. Network core timestamp
+may be slightly behind (usually 1 tick) due to latency introduced by the
+interrupt handling.
+
+For simplicity and low latency sample is not using more sophisticated IPM protocols.
+
+Building the application for nrf5340dk_nrf5340_cpuapp
+*****************************************************
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/nrf/nrf53_sync_rtc
+   :board: nrf5340dk_nrf5340_cpuapp
+   :goals: flash
+   :goals: flash --hex-file build/sync_rtc_net-prefix/src/sync_rtc_net-build/zephyr/zephyr.hex
```

Comment:
@mbolivar not sure if there is a better way to flash `ExternalProject`. In ninja i can type `ninja sync_rtc_net flash` but couldn't find west equivalent.